Arduino/Propulsion/src/PropulsionModule.cpp

```diff
@@ -139,6 +155,10 @@ void ResetModule() {
 }
 
 void GetConfig() {
+#ifdef TINYDUINO
+  errorByte = 0x03;
```

Comment:
I like this concept, and I think it would be a good idea to extend it to support the qik as well. That is, you can't get the config of a register that isn't a valid configuration register, so perhaps we can perform a check on the payload to see what register is being requested and compare it to see if it is valid, else return error 0x03.

Reply:
I agree that it can be extended to handle invalid requests for Qik registers but we can save that for a different PR to keep things localized here.

Reply:
Sounds like a plan. We'll add it to the next revision we do for the Qiks once we get to the stage where we are using the extra features.